README.md

```diff
@@ -1,64 +1,55 @@
 
 # PHPTAL - Template Attribute Language for PHP
 
-Master: [![Build Status](https://secure.travis-ci.org/pornel/PHPTAL.png?branch=master)](http://travis-ci.org/pornel/PHPTAL)
+[![Monthly Downloads](https://poser.pugx.org/phptal/phptal/d/monthly)](https://packagist.org/packages/phptal/phptal)
+[![License](https://poser.pugx.org/phptal/phptal/license)](https://packagist.org/packages/phptal/phptal)
+[![Build Status](https://secure.travis-ci.org/pornel/PHPTAL.png?branch=master)](http://travis-ci.org/pornel/PHPTAL)
+[![Scrutinizer Quality Score](https://scrutinizer-ci.com/g/phptal/PHPTAL/badges/quality-score.png)](https://scrutinizer-ci.com/g/phptal/PHPTAL/)
 
-Usage requirements
-==================
+Requirements
+============
 
 To use PHPTAL in your projects, you will only require PHP 5.1.2 or later.
 
-If you want to use the builtin internationalisation system (I18N) the gettext extension must be compiled into PHP (`--with-gettext`).
+If you want to use the builtin internationalisation system (I18N), the php-gettext extension must be installed or compiled into PHP (`--with-gettext`).
+
+Composer install (recommended)
+==============================
 
+You can install this package by using [Composer](http://getcomposer.org).
+Link to Packagist: https://packagist.org/packages/phptal/phptal
 
-Non-PEAR install
-================
+```
+{
+    "require": {
+        "phptal/phptal": "~1.3"
+    }
+}
+```
 
-To run you only need PHPTAL.php and files in PHPTAL directory. Other files are for unit tests and PEAR installer.
+Manual install
+==============
 
-Get the latest PHPTAL package from http://phptal.org.
+Get the latest PHPTAL tarball from https://github.com/phptal/PHPTAL/releases
 
     tar zxvf PHPTAL-X.X.X.tar.gz
     mv PHPTAL-X.X.X/PHPTAL* /path/to/your/php/include/path/
 
+Changelog
+=========
 
-PEAR Install
-============
-
-Get the latest PHPTAL package from http://phptal.org.
-
-Then run:
-
-    pear install PHPTAL-X.X.X.tar.gz
-
-
+[Please see the projects releases page](https://github.com/phptal/PHPTAL/releases)
```

Comment:
Note: need to makes sure we make these notes available as part of the repo as well:
 - as a file?
 - as the tag message (`git tag -F <some-release-notes>`)

This needs to be added to some contributing notes